subprojects/gradle/instrumentation-test-impact-analysis/src/main/kotlin/com/avito/instrumentation/impact/model/AffectionType.kt

```diff
@@ -7,7 +7,8 @@ internal enum class AffectionType {
     TEST_MODIFIED,
     TEST_RENAMED,
     /**
-     * Shows that test itself isn't changed, but changed classes in test source sets that are used byt test
+     * Shows that test itself isn't changed, but changed classes in test source sets that are used by test
      */
+    // use?
```

Comment:
What tests shall be that type? Should we run them like modified?

Reply:
That is out of scope for current task